firmware/broadcom_trx.ksy

```diff
@@ -1,6 +1,6 @@
 meta:
-  id: asus_trx
-  title: ASUS routers .trx official firmware packaging
+  id: broadcom_trx
+  title: Broadcom devices .trx firmware packaging
   file-extension: trx
   license: GPL-2.0
```

Comment:
When you are already editing the file, please also fix this deprecated SPDX identifier [`GPL-2.0`](https://spdx.org/licenses/GPL-2.0.html): is it supposed to be [`GPL-2.0-only`](https://spdx.org/licenses/GPL-2.0-only.html) or [`GPL-2.0-or-later`](https://spdx.org/licenses/GPL-2.0-or-later.html)?